### DISCUSSION TITLE
React Native 65 + AdMob Banner not working?

### DISCUSSION THREAD

**Discussion by Rebsos** [invertase/react-native-firebase#5730] [Q&A]
Is it compatible? It seems not to work together. I've tried it the whole day.

React Native: 0.65.1
React Native Firebase App + AdMob: 11.5.0

What I found out:
- In Production the Banner just doesn't get initialized and never appears
- No logs or errors from AdMob in Logcat. It seems not to send any request.
- Tried a new fresh project just with RN Firebase App + AdMob. Same.
- In Dev the Banner also doesn't get loaded, no logs in Logcat
- **BUT**, if you refresh (R) the App in DEV, it suddenly appears, with all logs in Logcat. So just on the first run or after the app is shutdown before, the Banner doesn't work. If you refresh the App, it works.
- I've delayed the Banner rendering for 3 Seconds (if there is a race), but it doesn'work
- admob().setRequestConfiguration()... runs always successfully, before Banner rendering.
- Downgrading targetSdk to 29 also doesn't work
- Downgrading to React Native 64, everything works fine!

**Comment by Rebsos** [comment]
Small update:
Interstitial Ads do work in dev and prod with RN 65 (Banners not). I've changed the title.

**Reply by mikehardy** [reply]
AdMob is no longer a part of Firebase at all upstream, so it's been removed from here as well, we're working on bringing the code that *was* here back to life as react-native-google-ads right now, please follow on #5329

**Reply by Rebsos** [reply]
@mikehardy thx for you response. I follow this project closely, but it's currently not ready. So I have to stay at 11.5 of RN-Firebase where AdMob is included

**Reply by mikehardy** [reply]
Yep - exactly that (staying on 11.5 for now) - react-native-google-ads is my current task across the various repos I collaborate with Invertase on so hopefully you'll have some good news soon
